api/main_endpoints/routes/User.js

```diff
@@ -55,7 +56,7 @@ router.post('/checkIfUserExists', (req, res) => {
 router.post('/delete', (req, res) => {
   if (!checkIfTokenSent(req)) {
     return res.sendStatus(FORBIDDEN);
-  } else if (!checkIfTokenValid(req)) {
+  } else if (!checkIfTokenValid(req, membershipState.OFFICER)) {
```

Comment:
I lack context here but it seems deleting a user should be reserved for Admin role. If officers have this privilege this is fine